docs/documentation.md

```diff
@@ -17,6 +17,111 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. Use the drop-down menu on the top right of any page to select the documentation version for the RabbitMQ release that you are using. We update each version of documentation with the latest minor and patch information for that release.
+
+For previous 3.12.x documentation, select the **3.12** option from the drop-down menu. 3.12.x documentation is hosted on our previous RabbitMQ site. This documentation is not versioned. 
+
+Use the navigation on the left to browse through documentation for your release of RabbitMQ.
+
+## Getting Started
+Choose which RabbitMQ version you want to use, review the [latest RabbitMQ releases](https://revamp-the-rabbitmq-home-pag.rabbitmq-website.pages.dev/docs/changelog) to help you decide. Next, go to the [Get Started pages](https://revamp-the-rabbitmq-home-pag.rabbitmq-website.pages.dev/tutorials) to try out the tutorials. These tutorials offer an introduction to messaging and it's key features as well as some common usage scenarios.
+
+## User: Developer
+
+For a developer who wants to use RabbitMQ in an application, the following is the generic list of steps they would follow to start using RabbitMQ.
+
+<table id="developer">
+  <tr>
+    <th class="centre">Task</th>
+    <th class="centre">Documentation Reference</th>
+  </tr>
+
+  <tr>
+    <td class="centre"></td>
+    <td class="centre">XXX</td>
+  </tr>
+
+  <tr>
+    <td class="centre"></td>
+    <td class="centre">XXX</td>
+  </tr>
+
+  <tr>
+    <td class="centre"></td>
+    <td class="centre">XXX</td>
+  </tr>
+</table> 
+
+## User: Administrator
+
+For an administrator who want to use RabbitMQ in a production environment, the following is the generic list of tasks they would follow to start using RabbitMQ.
+
+<table id="administrator">
+   <tr>
+    <th class="centre">Tasks</th>
+    <th class="centre">Documentation Reference</th>
+  </tr>
+  <tr>
+    <td class="centre">Install and provision RabbitMQ on all Nodes in the Cluster</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Configure the Hostnames</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Configure High Availability (if required)</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Configure exchanges, routing, and bindings</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Choose the appropriate queue to use (quorum queues, classic queues, or streams) and configure these Queues</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Install and configure additional plugins</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Choose the appropriate Queue type to use and configure the Queues</td>
+    <td class="centre">XXX</td>
+  </tr>
+  <tr>
+    <td class="centre">Configure publishers and consumers</td>
```

Comment:
Administrators do not configure publishers and consumers, applications do, at least applications do the most important parts. This belongs to the developer section.